BUILD.bazel

```diff
@@ -21,6 +21,8 @@ exports_files([
 ])
 
 EXPERIMENTAL_LIBRARIES = [
+    # Introduced circa 2022-09-07
+    "networkconnectivity",
```

Comment:
Arguably this library could go straight to GA: go/client-quality#major-release-requirements

Reply:
Done. I regrettably decided to just reset the commits instead of generating the GA scaffolding over the manual changes.